apps/server/src/form-instances/form-instances.service.ts

```diff
@@ -99,4 +99,31 @@ export class FormInstancesService {
       },
     });
   }
+
+  async signFormInstance(formInstanceId: string, signatureId: string) {
+    const formInstance = await this.prisma.formInstance.findUnique({
+        where: { id: formInstanceId },
+        include: { signatures: { include: { signerPosition: true, userSignedBy: true } } },
+    });
+
+    if (!formInstance) {
```

Comment:
let's use the defined form instance error message, and throw a NotFoundException instead here